index.js

```diff
@@ -0,0 +1,27 @@
+const menu = document.querySelector('.burger-menu');
+const closeItem = document.querySelector('.close-item');
+const aboutMenu = document.querySelector('.about-menu');
+const programMenu = document.querySelector('.porgram-menu');
+const speakersMenu = document.querySelector('.speak-menu');
+const partnersMenu = document.querySelector('.partners-menu');
+const contactMenu = document.querySelector('.contact-menu');
+const conferenceMenu = document.querySelector('.conference-menu');
+
+function hideMenu() {
+  const mobileMenu = document.querySelector('.mobile-menu');
+  mobileMenu.style.display = 'none';
+}
+function showMenu(e) {
+  e.preventDefault();
+  const mobileMenu = document.querySelector('.mobile-menu');
+  mobileMenu.style.display = 'flex';
+}
```

Comment:
- [ ] Kindly replace these `regular` functions with the `arrow` function. In the [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) it's required to use es6 syntax.

Reply:
Alright, thank you!

Reply:
This issue is well resolved! ✔️